bundle.js

```diff
@@ -8,16 +8,68 @@
   var require_newsApi = __commonJS({
```

Comment:
During one of the reviews, another cohort member mentioned that apparently it's a convention to abstract bundle.js, so it's worth adding the bundle.js file into our .gitignore file as well.